src/util/scoring/css.rs

```diff
@@ -89,5 +94,24 @@ pub async fn render_diff_image(
 
 #[allow(unused_variables)]
 pub async fn execute(code: &str, template: Template) -> anyhow::Result<ExecutionResult> {
-    todo!()
+    
+    // Not existed in local
+    if metadata(&template.local_path).is_err() {
+        let hub = drive::HubDrive::new().await?;
```

Comment:
Use once_cell::Lazy to init the hub instead of re-init everytime need to access drive